lib/src/material_controls.dart

```diff
@@ -397,3 +444,50 @@ class _MaterialControlsState extends State<MaterialControls> {
     );
   }
 }
+
+class _PlaybackSpeedDialog extends StatelessWidget {
```

Comment:
We can't share them because their implementation is different. Or did you mean make separate file like "cupertino_speed_dialog" and "material_speed_dialog" just like "cupertino_progress_bar" and "material_progress_bar"?